docs/content/documentation/content/page.md

```diff
@@ -3,29 +3,48 @@ title = "Page"
 weight = 30
 +++
 
-A page is any file ending with `.md` in the `content` directory, except files
-named `_index.md`.
-
-If a file ending with `.md` is named `index.md`, then it will generate a page
-with the name of the containing folder (for example, `/content/about/index.md` would
-create a page at `[base_url]/about`).  (Note the lack of an underscore; if the file
-were named `_index.md`, then it would create a **section** at `[base_url]/about`, as
-discussed in the prior part of this documentation.  But naming the file `index.md` will
-create a **page** at `[base_url]/about`).
-
-If the file is given any name *other* than `index.md` or `_index.md`, then it will
-create a page with that name (without the `.md`). So naming a file in the root of your
-content directory `about.md` would also create a page at `[base_url]/about`.
-Another exception to that rule is that a filename starting with a datetime (YYYY-mm-dd or [a RFC3339 datetime](https://www.ietf.org/rfc/rfc3339.txt)) followed by
-an underscore (`_`) or a dash (`-`) will use that date as the page date, unless already set
-in the front-matter. The page name will be anything after `_`/`-` so a filename like `2018-10-10-hello-world.md` will
-be available at `[base_url]/hello-world`. Note that the full RFC3339 datetime contains colons, which is not a valid
-character in a filename on Windows.
-
-As you can see, creating an `about.md` file is exactly equivalent to creating an
-`about/index.md` file.  The only difference between the two methods is that creating
-the `about` folder allows you to use asset colocation, as discussed in the
-[Overview](@/documentation/content/overview.md#assets-colocation) section of this documentation.
+A page is any file ending with `.md` in the `content` directory, except files named `_index.md` because these produce sections and not pages.
+
+Creating an `about.md` file is exactly equivalent to creating an `about/index.md` file.  The only difference between the two methods is that creating the `about` folder allows you to use asset colocation, as discussed in the [Overview](@/documentation/content/overview.md#assets-colocation) section of this documentation.
+
+## Output paths
+
+For any page within your content folder, its output path will be defined by either:
+
+- its `slug` frontmatter key
+- its filename
+
+Either way, these proposed path will be sanitized before being used. If `slugify_paths` is enabled in the site's config, paths are [slugified](https://docs.rs/slug/0.1.4/slug/fn.slugify.html). Otherwise, a simpler sanitation is performed, outputting only valid NTFS paths. The following characters are removed: `<`, `>`, `:`, `/`, `|`, `?` `*` `#` `\\`, as well as newlines and tabulations. Additionally, trailing whitespace and dots are removed.
```

Comment:
I would link to wikipedia https://en.wikipedia.org/wiki/Clean_URL#Slug rather than the slug crate docs